packages/clay-color-picker/src/Hue.tsx

```diff
@@ -20,66 +20,56 @@ type Props = {
 	value: number;
 };
 
-const useIsomorphicLayoutEffect =
-	typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect;
-
 /**
  * Renders Hue component
  */
 const ClayColorPickerHue = ({value = 0, onChange = () => {}}: Props) => {
 	const containerRef = React.useRef<HTMLDivElement>(null);
-	const selectorActive = React.useRef<boolean>(false);
 
-	const {onPointerMove, setXY, x, y} = usePointerPosition(containerRef);
+	const {setXY, y} = usePointerPosition(containerRef);
 
-	const removeListeners = () => {
-		selectorActive.current = false;
+	const [internalValue, setInternalValue] = useState<number>(value);
```

Comment:
We don't need another state here, because we also have a state in the parent component, in which case we have access to them which are `value` and `onChange` it is easier to maintain a source of truth than to have multiple states and keep synchronizing them all, this is also bad because of unnecessary rerenders.